content/en/docs/releases/supported-releases/index.md

```diff
@@ -79,7 +79,7 @@ Please keep up-to-date and use a supported version.
 
 | Minor Releases   | Patched versions with no known CVEs           |
 |------------------|-----------------------------------------------|
+| 1.16.x           | 1.16.0+                                       |
```

Comment:
we already released 1.16.1 , so should this be 1.16.1+ ?

Reply:
1.16.0 doesn't have any CVE's so it's still 1.16.0. We only increase this value when we have an actual CVE. Example: 1.15.3 was determined as https://istio.io/latest/news/security/istio-security-2022-008/ says 1.15.2 has a CVE. 1.14 was determined by https://istio.io/latest/news/security/istio-security-2022-007/.